mlflow/models/evaluation/base.py

```diff
@@ -335,6 +341,28 @@ def baseline_model_metrics(self) -> Dict[str, Any]:
         """
         return self._baseline_model_metrics
 
+    @experimental
+    @property
+    def table(self) -> Dict[str, "pd.DataFrame"]:
+        """
+        A dictionary mapping standardized artifact names (e.g. "eval_results_table") to
+        corresponding table content as pandas DataFrame.
+        """
+        eval_table = {}
+        if self._run_id is None:
+            _logger.warning("Cannot load eval_results_table because run_id is not specified.")
+            return eval_table
+
+        for table_name, table_path in self._artifacts.items():
+            path = urllib.parse.urlparse(table_path.uri).path
+            table_fileName = os.path.basename(path)
+            try:
+                eval_table[table_name] = mlflow.load_table(table_fileName, run_ids=[self._run_id])
+            except Exception:
+                pass  # Swallow the exception since we assume its not a table.
+
```

Comment:
should we emit some logger message here?

Reply:
maybe for debugging but for end user any message might be confusing as to why asking for table returned back a log message. Its an implementation detail that is unfortunate that we don't have a way to distinguish artifacts that are table v/s other artifacts :(